Brokerages/Fxcm/FxcmBrokerage.DataQueueHandler.cs

```diff
@@ -61,18 +64,16 @@ public IEnumerator<BaseData> Subscribe(SubscriptionDataConfig dataConfig, EventH
         /// Adds the specified symbols to the subscription
         /// </summary>
         /// <param name="symbols">The symbols to be added keyed by SecurityType</param>
-        private void Subscribe(IEnumerable<Symbol> symbols)
+        /// <param name="tickType">Type of tick data</param>
+        private bool Subscribe(IEnumerable<Symbol> symbols, TickType tickType)
         {
-            var symbolsToSubscribe = (from symbol in symbols 
-                                      where !_subscribedSymbols.Contains(symbol) && CanSubscribe(symbol)
-                                      select symbol).ToList();
-            if (symbolsToSubscribe.Count == 0)
-                return;
+            if (symbols.Any())
+                return false;
```

Comment:
This should be `!symbols.Any()`? not sure this is a possible case though? Same for `Unsubscribe`

Reply:
done